### PR TITLE
Adding Quebec City

### PR DESCRIPTION
**Pull request by scouturier**

### CODE REVIEW

**Comment by mathieu-benoit** [comment]
Hi @iennae, thanks for pointing this out, but our event will be in 2020, we are working in a draft for now as `[WIP]`. We will let you know when it will be ready for your review, thanks!

File: content/events/2020-quebeccity/_index.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+---
+title: Québec
+date: '2020-06-11'
+end: '2021-06-18'
```
Comment (iennae) [inline content/events/2020-quebeccity/_index.md]: This should be changed to 2020 :)

Comment (mathieu-benoit) [reply content/events/2020-quebeccity/_index.md]: Good catch, thanks and sorry! :)

Comment (mathieu-benoit) [reply content/events/2020-quebeccity/_index.md]: Done, thanks!

File: content/events/2020-quebeccity/_index.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    url: https://mayadata.io/
+  - name: Lecentre.net
+    logo: sponsors/lecentrenet.png
+    url: http://www.lecentre.net/
+speakers:
+- name: Phippy
+  image: speakers/phippy-color.png
+  twitter: https://twitter.com/cloudnativefdn
+  bio: Phippy is a simple PHP app, trying to find a home in a cloud native world.
+  homepage: https://phippy.io
```
Comment (iennae) [inline content/events/2020-quebeccity/_index.md]: This should just get commented out or removed. You don't need to have speakers listed right now.

File: content/events/2020-quebeccity/_index.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+  homepage: https://www.linkedin.com/in/martin-guy-lapointe-37aa37b4/
+- name: Ayrat Khayretdinov
+  image: organizers/ayrat.jpg
+  #twitter: https://twitter.com/cloudnativefdn
+  homepage: https://www.linkedin.com/in/ayrat-khayretdinov-8503b6b/
+organizers_email: organizers-quebec-city@kubernetescommunitydays.org
+local_law_enforcement: 911
+medical_emergency: 911
+cfp_deadline: '2020-05-15'
+sponsor_deadline: '2020-02-15'
```
Comment (iennae) [inline content/events/2020-quebeccity/_index.md]: These deadlines seem pretty short for sponsor, and potentially long for the CFP. Will you have time to review proposals and get back to folks for acceptances before event?

**Comment by dankohn** [comment]
6.4 MB is *way* too big for a JPEG image. @KTan1226 Could you please add a checkbox to the pull request template saying that all images should be below 500kB so as not to slow down page load times.

**Comment by mathieu-benoit** [comment]
Hi @dankohn @KTan1226 and @iennae, we think we are good to proceed to publish this website in this state. We have taken into consideration your remarks and we think we are good with the dates indicated. Please do let us know if you have any questions for us. Thanks.

Note: we will open later another PR when we will be ready about CFP and EventBrite.

File: content/events/2020-quebeccity/_index.md (hunk 2 above)
Comment (mathieu-benoit) [reply content/events/2020-quebeccity/_index.md]: Done, thanks!

File: content/events/2020-quebeccity/_index.md (hunk 3 above)
Comment (mathieu-benoit) [reply content/events/2020-quebeccity/_index.md]: Thanks for pointing this out, we reviewed those dates with the team and we are comfortable with them for our local edition here. Thanks.

File: content/events/2020-quebeccity/_index.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+location:
+  countrycode: ca
+  country: Canada
+  city: Québec
+  url: https://goo.gl/maps/xTvcknrmbL5fQDEcA
+  venue: ENAP
+map: enap.PNG
+sched_slug: kubernetescommunitydayquebeccity
+social:
+  #twitter: 'https://twitter.com/Bouvet_3Y0Z'
```
Comment (KTan1226) [inline content/events/2020-quebeccity/_index.md]: Did you create a Twitter handle for your event? If so, can you please update here? Thanks!

Comment (mathieu-benoit) [reply content/events/2020-quebeccity/_index.md]: Hi @KTan1226, we don't have such Twitter handle for the event. Thanks.

**Comment by mathieu-benoit** [comment]
Hi @dankohn @KTan1226 and @iennae, just curious to know if there is any chance to get this PR merged today? We would like to proceed with some announcements around this. Thanks!

**Pull request merged**